conn.go

```diff
@@ -729,3 +727,20 @@ func (c *Conn) SendKeyUpdate(requestUpdate bool) error {
 func (c *Conn) GetHsState() string {
 	return reflect.TypeOf(c.hState).Name()
 }
+
+func (c *Conn) ComputeExporter(label string, context []byte, keyLength int) ([]byte, error) {
```

Comment:
Just to confirm: the plan is to add a `ComputeEarlyExporter` function for the early exporter and then to factor out the necessary parts of this function.